### DISCUSSION TITLE
Support newer image classifiers for labels

### DISCUSSION THREAD

**Discussion by Radtoo** [photoprism/photoprism#3590] [Ideas]
Many of the newer open source image classifiers are vastly better than the labeling currently available in Photoprism.

Examples: https://github.com/xinyu1205/recognize-anything, https://huggingface.co/spaces/nielsr/comparing-captioning-models . Even anime trained models like https://huggingface.co/7eu7d7/ML-Danbooru / https://huggingface.co/SmilingWolf are far better. Support for any of these could very much increase the accuracy and usefulness of labels. 

Ideally allow users pick 1-n classifier models to generate tags for (each maintaining their own db table) & have the classifier or confidence threshold for each filterable in the Webui. It will make it a lot less necessary to standardize tags via mappings.

**Comment by yggdrasil75** [comment]
Please update to a newer model. BLIP, CLIP, VIT, so on. this is something that the program desperately needs, having a signpost be classified as a "bench" makes me never want to use label search.
